lib/beaker/hypervisor/docker.rb

```diff
@@ -4,6 +4,7 @@ class Docker < Beaker::Hypervisor
     def initialize(hosts, options)
       require 'docker'
       @options = options
+      @options[:set_env] = false
```

Comment:
I'm not a big fan of this sort of hidden override (ie, we'll listen to your option, unless you run docker and then we'll ignore you).  Any reason why you can't just override with your desired set_env value in your host configuration file?